Move project page link to admin submission page

devkonrad opened the pull request:
Move project page link to admin submission page

Changes:
modified: app/data_grids/submissions_grid.rb

Refs: https://github.com/Iridescent-CM/technovation-app/issues/3417

viviancan reviewed (changes requested):
Please view the comment from Deb in #3417 for the updated changes! Thx!

devkonrad commented:
> Please view the comment from Deb in #3417 for the updated changes! Thx!

Thanks vivian. Just fix it! :)

devkonrad commented:
> Thanks for the changes, Alex!
> 
> I'm not 100% sure what is going on, but with the "export csv" option, the project page link is not part of the csv file! This should be addressed before merging to QA. Thanks!

Thamks for the catch, @viviancan! I will toake a look here :)

devkonrad commented on app/data_grids/submissions_grid.rb:
```diff
@@ -60,6 +60,23 @@ class SubmissionsGrid
+
+  column :project_page, html: false do |sub|
+    url_helper = Rails.application.routes.url_helpers
+    "https://my.technovationchallenge.org#{url_helper.project_path(sub)}"
```
I put the URL encoded, because I couldn't find a way to put the dynamics in the csv. Since the request object is unavailable (if anyone knows how to do it dynamically, I'd appreciate it :))

shaunxp20 replied:
Try this to see if this is what we want:

```
column :project_page, html: false do |submission|
  Rails.application.routes.url_helpers.url_for(controller: "projects", action: "show", id: submission)
end
```

devkonrad replied:
> Try this to see if this is what we want:
> 
> ```
> column :project_page, html: false do |submission|
>   Rails.application.routes.url_helpers.url_for(controller: "projects", action: "show", id: submission)
> end
> ```

Thanks, @shaunxp20 ! This works perfectly!

devkonrad commented:
> Thanks for the changes, Alex!
> 
> I'm not 100% sure what is going on, but with the "export csv" option, the project page link is not part of the csv file! This should be addressed before merging to QA. Thanks!
I updated the PR and fixed this thing. :)

viviancan reviewed (approved):
Verifying that I was able to successfully export the submissions csv and the project page URL was there! Thx Alex
